### PR TITLE
DM-29671: Add DP0.1 data choice

### PR DESCRIPTION
**Pull request by hsinfang**

### CODE REVIEW

File: dp0.tex [hunk 1]
```diff
@@ -36,7 +36,7 @@ \subsection{Elements of Data Preview 0.1}
 
 \subsubsection {Dataset choice considerations} \label{sec:dataset}
 
-For DP0.1 we are moving ahead with DESC DR6 as the baseline. We may still load HSC at some point.
+For DP0.1 we are moving ahead with DESC DR6 WFD as the baseline. We may still load HSC at some point.
```
Comment (gpdf) [inline dp0.tex]: "At some point" _in DP0.1_ or later on, in the DP0.2 time frame?

File: dp0.tex [hunk 2]
```diff
@@ -36,7 +36,7 @@ \subsection{Elements of Data Preview 0.1}
...
 
 
 We do no have  HiPS maps available for DP0.1.
```
Comment (gpdf) [inline dp0.tex]: "We do not"

(Btw: I am working on this, but there are no guarantees yet.)

File: dp0.tex [hunk 3]
```diff
@@ -56,13 +56,38 @@ \subsubsection{Data Products Offered}
...
+
+
+In DP0.2, the exact science data products depend on what pipelines are ready for our reprocessing.
 
 Questions:
 
 \begin{itemize}
 
-\item Are we offering parquet files? --- No promise. Currently our SDMified parquet-generating pipelines are HSC only and Gen2 only. If parquet files are offered the access will be via the read-only Butler Gen3 repo.
+\item Are we offering parquet files? --- No in DP0.1; likley yes in DP0.2. Currently our SDMified parquet-generating pipelines are HSC only and Gen2 only. If parquet files are offered the access will be via the read-only Butler Gen3 repo.
```
Comment (gpdf) [inline dp0.tex]: "likely"

Also, "Parquet" is how Apache writes the name.

File: dp0.tex [hunk 4]
```diff
@@ -56,13 +56,38 @@ \subsubsection{Data Products Offered}
...
 
 \item We should presumably explicitly rule out bulk download  --- YES. However, this ({\it was} discussed at the 2019 PCW, as a potential mitigation against there not being batch compute available in DP0. If a particular group requested bulk download, it could be an opportunity to start developing that capability. We will also need to know whether to allow DESC bulk download access as part of the MOU to gain access to DC2: they may well want to download all the re-processed products, for their own purposes (and to develop their capability to ingest and work with bulk downloads).
```
Comment (gpdf) [inline dp0.tex:95]: I would feel more comfortable if this paragraph definitively ruled it out for DP0.1.  It could still be left as TBD for DP0.2.

**Review by gpdf — COMMENTED** [review]
Very useful update, thank you!

File: dp0.tex (hunk 1 above)
Comment (hsinfang) [reply dp0.tex]: No within the DP0.2 time frame, but maybe later.  Currently there isn't a really plan loading HSC data, as far as I know.

File: dp0.tex (hunk 4 above)
Comment (hsinfang) [reply dp0.tex:95]: I didn't write this paragraph but I interpreted the capital "YES" as definitely ruling it out for DP0. Do you mean you want the statement to be even stronger?

File: dp0.tex (hunk 1 above)
Comment (gpdf) [reply dp0.tex]: Then I would say "We may still load Rubin-processed HSC data at some later point in the DP0 era."

File: dp0.tex (hunk 4 above)
Comment (gpdf) [reply dp0.tex:95]: Well, it's kind of stream-of-consciousness the way it reads.  "YES." followed by "However," sounds a lot like "Well, not really" to me.

Maybe "We have ruled out offering bulk download facilities for DP0.1.  They are unlikely to be available at any time during the DP0 era, but we are still evaluating that against the time scale of the Data Previews and the schedule for the remaining required development effort.  Making bulk download available had previously been discussed as a possible mitigation for the absence of user-facing batch compute in DP0.  We still need to determine whether we need to provide some form of bulk access to DESC to return the Rubin-processed DC2 data from DP0.2 to them for their own use, both for scientific purposes and to allow them to develop their capability to ingest and work with bulk downloads."

@leannep Any thoughts on this?

File: dp0.tex (hunk 1 above)
Comment (hsinfang) [reply dp0.tex]: Re-reading it, I find mentioning HSC here confusing.  So I'm removing this sentence, and referring readers to the Appendix where some HSC data consideration is discussed.

File: dp0.tex (hunk 4 above)
Comment (hsinfang) [reply dp0.tex:95]: I see what you mean now. What you suggested sounds clearer and it's in the commit now.

File: dataset.tex [hunk 5]
```diff
@@ -1,5 +1,5 @@
 
-\section {Dataset choice considerations} \label{sec:dataset}
+\section {Dataset choice considerations} \label{sec:datasetHistory}
 
 {\bf For DP0 we are moving ahead with DESC DR6 as the baseline.}
 This seciton is included for historical context on the decision making.
```
Comment (leannep) [inline dataset.tex]: This section (spelling)

File: dp0.tex [hunk 6]
```diff
@@ -56,15 +57,41 @@ \subsubsection{Data Products Offered}
...
+\begin{itemize}
+\item \texttt{position}: DESC internal data as ingested in DESC's Qserv instance at IN2P3.
+\item \texttt{dpdd\_ref}: DESC internal data as ingested in DESC's Qserv instance at IN2P3.
+\item \texttt{dpdd\_forced}: DESC internal data as ingested in DESC's Qserv instance at IN2P3.
+\item \texttt{object}: DESC internal data v2 at NERSC provided to us in parquet format.
+\item \texttt{truth\_match}: DESC internal data v2 at NERSC provided to us in parquet format.
+\end{itemize}
+
+The science data products in the Butler Gen3 repository depend on the availability in the DESC-provided Gen2 data repository and the Gen2-to-3 conversion.
+Besides raw data, the following 4 reruns were obtained from DESC's copy at IN2P3.
```
Comment (leannep) [inline dp0.tex]: in addition to the raw data , ....

File: dp0.tex [hunk 7]
```diff
@@ -56,15 +57,41 @@ \subsubsection{Data Products Offered}
 The latter is closer to the eventual data access but the former allows additional scientific analysis.
 We will provide both in DP0.1. (See milestones L3-MW-0010 and L3-MW-0020 for Qserv loading)
 
-The exact science data products depend on what exist in the provided data repository (if serving existing data products) or what pipelines are ready for our reprocessing.
+Specifically, in DP0.1, the following tables from DESC will be provided as one database \texttt{dp01\_dc2\_catalogs} in one TAP schema:
```
Comment (leannep) [inline dp0.tex]: in one TAP schema in Rubin's Qserv instance running at the IDF.

File: dp0.tex [hunk 8]
```diff
@@ -56,15 +57,41 @@ \subsubsection{Data Products Offered}
...
+
+
+In DP0.2, the exact science data products depend on what pipelines are ready for our reprocessing.
 
 Questions:
 
 \begin{itemize}
 
-\item Are we offering parquet files? --- No promise. Currently our SDMified parquet-generating pipelines are HSC only and Gen2 only. If parquet files are offered the access will be via the read-only Butler Gen3 repo.
+\item Are we offering Parquet files? --- No in DP0.1; likely yes in DP0.2. Currently our SDMified Parquet-generating pipelines are HSC only and Gen2 only. If Parquet files are offered the access will be via the read-only Butler Gen3 repo.
```
Comment (leannep) [inline dp0.tex]: likely yes in DP0.2. -> possibly in DP0.2

File: dp0.tex [hunk 9]
```diff
@@ -56,15 +57,41 @@ \subsubsection{Data Products Offered}
...
+\item We should presumably explicitly rule out bulk download  --- YES.
+We have ruled out offering bulk download facilities for DP0.1.  They are unlikely to be available at any time during the DP0 era, but we are still evaluating that against the time scale of the Data Previews and the schedule for the remaining required development effort. Making bulk download available had previously been discussed as a possible mitigation for the absence of user-facing batch compute in DP0. We still need to determine whether we need to provide some form of bulk access to DESC to return the Rubin-processed DC2 data from DP0.2 to them for their own use, both for scientific purposes and to allow them to develop their capability to ingest and work with bulk download.
```
Comment (leannep) [inline dp0.tex]: I agree with this position on bulk download. We will not test the RSP  if people just download data. DESC DC2 data are public now, if people want to bulk download the data, they can get this dataset elsewhere. 

I would  remove everything after "We have ruled out offering bulk download facilities for DP0.1."  and add "The DESC DC2 dataset is public and can be downloaded from https://lsstdesc-portal.nersc.gov/."

File: dp0.tex (hunk 4 above)
Comment (leannep) [reply dp0.tex:95]: Just added a comment above before seeing this, essentially to the same effect. I personally would leave all bulk download until DP1. (i.e LSST data). DESC DC2 data are now public and those who want to bulk download can go to the DESC portal. We have plenty of time to test bulk download from DP1 onwards.  We are planning to provide batch compute in DP0.2 I think?

**Review by leannep — CHANGES_REQUESTED** [review]

File: dp0.tex (hunk 4 above)
Comment (hsinfang) [reply dp0.tex:95]: No, we won't provide user batch compute in DP0.2.

File: dp0.tex (hunk 9 above)
Comment (hsinfang) [reply dp0.tex]: Sounds good. Done.

**Comment by gpdf** [comment]
@leannep The remaining bulk download question was, I think, originally about whether we felt we had some kind of moral obligation to DESC to return the _outputs_ of our DP0.2-era processing of DC2 to them for future use.  That could be done "privately" if we feel we must, rather than by exercising the "real" bulk download facility, or it could be an opportunity to do that exercise.  Or we could just not do it.  :)

File: dp0.tex [hunk 10]
```diff
@@ -56,16 +57,42 @@ \subsubsection{Data Products Offered}
...
 Questions:
 
 \begin{itemize}
 
-\item Are we offering parquet files? --- No promise. Currently our SDMified parquet-generating pipelines are HSC only and Gen2 only. If parquet files are offered the access will be via the read-only Butler Gen3 repo.
-
-\item We should presumably explicitly rule out bulk download  --- YES. However, this ({\it was} discussed at the 2019 PCW, as a potential mitigation against there not being batch compute available in DP0. If a particular group requested bulk download, it could be an opportunity to start developing that capability. We will also need to know whether to allow DESC bulk download access as part of the MOU to gain access to DC2: they may well want to download all the re-processed products, for their own purposes (and to develop their capability to ingest and work with bulk downloads).
+\item Are we offering Parquet files? --- No in DP0.1; possibly in DP0.2. Currently our SDMified Parquet-generating pipelines are HSC only and Gen2 only. If Parquet files are offered the access will be via the read-only Butler Gen3 repo.
 
+\item We should presumably explicitly rule out bulk download  --- YES.
```
Comment (womullan) [inline dp0.tex]: Lets remove this question line and just keep the statement.

**Review by womullan — APPROVED** [review]
looks good thank you

**Comment by leannep** [comment]
> @leannep The remaining bulk download question was, I think, originally about whether we felt we had some kind of moral obligation to DESC to return the _outputs_ of our DP0.2-era processing of DC2 to them for future use. That could be done "privately" if we feel we must, rather than by exercising the "real" bulk download facility, or it could be an opportunity to do that exercise. Or we could just not do it. :)

Let's talk to DESC - I think doing this privately is a good option and would not be too burdensome, if DESC want it.

**Review by gpdf — APPROVED** [review]
Looks good now!

**Pull request merged**